src/ng/compile.js

```diff
@@ -186,6 +186,27 @@
  *   you want to shallow watch for changes (i.e. $watchCollection instead of $watch) you can use
  *   `=*` or `=*attr` (`=*?` or `=*?attr` if the property is optional).
  *
+ * * `<` or `<attr` - set up one-way (one-directional) binding between a local scope property and the
```

Comment:
Having this connotation is not as important as **not having** the opposite. As for multi-character sigils, I think they would look too scary in combination with `?` and `*`:
```js
bindings: { propertyName: "<=*?attrName" } // oh...
```